packages/components/src/components/toolbar/shadow.tsx

```diff
@@ -0,0 +1,159 @@
+import type { JSX } from '@stencil/core';
+import { Component, Element, h, Host, Listen, Prop, State, Watch } from '@stencil/core';
+
+import type { LabelPropType, ToolbarAPI, ToolbarStates, ToolbarItemsPropType, ToolbarItemPropType } from '../../schema';
+import { validateLabel, validateToolbarItems } from '../../schema';
+import { KolLinkTag, KolButtonTag } from '../../core/component-names';
+
+const TOOLBAR_ITEM_TAG_NAME = 'kol-toolbar-item';
+
+@Component({
+	tag: 'kol-toolbar',
+	styleUrls: {
+		default: './style.scss',
+	},
+	shadow: true,
+})
+export class KolToolbar implements ToolbarAPI {
+	@Element() host!: HTMLElement;
+
+	@State() public state: ToolbarStates = {
+		_label: '',
+		_items: [],
+	};
+
+	@State() private currentIndex: number = 0;
+
+	private renderElement = (element: ToolbarItemPropType, index: number): JSX.Element => {
```

Comment:
```suggestion
	private renderItem = (element: ToolbarItemPropType, index: number): JSX.Element => {
```
Dann ist es konsistent mit der übrigens Benennung :)